tests/test-content/js/layout/placeholder.js

```diff
@@ -0,0 +1,36 @@
+// Copyright 2018 Endless Mobile, Inc.
+
+/* exported Placeholder */
+
+const GObject = imports.gi.GObject;
+const Gtk = imports.gi.Gtk;
+
+const Config = imports.app.config;
+const Module = imports.app.interfaces.module;
+
+/**
+ * Class: Placeholder
+ * This is a modified version of the SDK Placeholder version
+ */
+var Placeholder = new Module.Class({
```

Comment:
I think you could make this inherit from Layout.Placeholder and just always set the "Overridden" label in the constructor.

Reply:
Well, the trick that allows the "override" to happen is simply the fact that app-version file takes precedence  over the SDK-version file, so (as of now) it would be like trying to import itself (which won't work).

Reply:
In that case - it doesn't seem all that useful to override a framework module, if it's forced to be a completely unrelated implementation. Are you sure that we want to encourage that?

Reply:
Like I mentioned before, this "feature" is the side effect of merging both framework's and app's modules into the same importer node 'imports.framework.modules.*'.

The only benefit I see from this "feature" is to implement customized modules without the extra step of using custom YAML (e.g, to reference the new custom module).

But, yeah, this could be a double-edge sword as tool, e.g. in case some developer inadvertently clashes with framework's modules. 

It could be that the risk is higher than the benefit, and since we don't really loose any functionalities, we could just prevent this.